logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/featureConfig/GetRemoteFileSharingStatusAndPersistUseCase.kt

```diff
@@ -5,69 +5,64 @@ import com.wire.kalium.logic.configuration.UserConfigRepository
 import com.wire.kalium.logic.data.featureConfig.FeatureConfigModel
 import com.wire.kalium.logic.data.featureConfig.FeatureConfigRepository
 import com.wire.kalium.logic.feature.user.IsFileSharingEnabledUseCase
+import com.wire.kalium.logic.featureFlags.KaliumConfigs
 import com.wire.kalium.logic.functional.fold
+import com.wire.kalium.logic.kaliumLogger
 import com.wire.kalium.network.exceptions.KaliumException
 import com.wire.kalium.network.exceptions.isNoTeam
 
 /**
  * This use case is to get the file sharing status of the team management settings from the server and
  * save it in the local storage (in Android case is shared preference)
  */
-interface GetRemoteFeatureConfigStatusAndPersistUseCase {
-    suspend operator fun invoke(): GetFeatureConfigStatusResult
+internal interface GetRemoteFeatureConfigStatusAndPersistUseCase {
```

Comment:
Can we rename this to `SyncFeatureConfigsUseCase` and `SyncFeatureConfigsUseCaseImpl` so it matches the other UseCases being called during SlowSync?